modules/ROOT/pages/am-installing.adoc

```diff
@@ -58,23 +58,16 @@ Ensure that the server has at least the minimum hardware requirements for Anypoi
 
 == Install Anypoint Monitoring
 
-Instructions for downloading and installing Anypoint Monitoring for an on-premises installation are located in the *Settings* section of the Anypoint Monitoring user interface.
-
-[NOTE]
-You must set the *Environment* and *Server* properties before you download the `am.zip`.
-
-== Access Settings
+Instructions for downloading and installing Anypoint Monitoring for an on-premises installation are located on the *Settings* page of the Anypoint Monitoring user interface.
```

Comment:
Bit hard to read this. How about removing "for an" and "installation" to have: 
"Instructions for downloading and installing Anypoint Monitoring on-premises are located on the *Settings* page of the Anypoint Monitoring user interface.

Reply:
Not sure if this should be "on-premises" or "on premises", but which way you choose, please make it the same as below.